src/OperatorExpression.cpp

```diff
@@ -127,7 +122,7 @@ std::unique_ptr<ExpressionInterface<T>> OperatorExpression<T>::
 template<typename T>
 std::string OperatorExpression<T>::toString() const
 {
-    return "(" + left_->toString() + " " + operatorGlyph_ + " " + 
+    return "(" + left_->toString() + " " + char(operatorType_) + " " + 
```

Comment:
I'm not sure what the coding guidelines say, I've mostly seen c-style casts like `(char) operatorType_` instead of the function-like notation you are using.

I think regardless c++-style static_cast is preferred, i.e. `static_cast<char>(operatorType_)`

not going to force you to implement this feedback - just food for thought